internal/tools/orchestrator/scheduler/executor/plugins/k8s/service.go

```diff
@@ -153,6 +152,12 @@ func newService(service *apistructs.Service, selectors map[string]string) *apiv1
 		}
 	}
 
+	err := setCoreErdaLabels(nil, service, k8sService.Labels)
```

Comment:
That will create two different Service resources, each used for all runtimes with the same name, as well as runtimes below a specified service group. We need different labels to distinguish between them.
e.g.
```shell
demo                             ClusterIP   172.21.46.129   <none>        8080/TCP,7098/TCP   43h
demo-9230e77cd4                  ClusterIP   172.21.41.203   <none>        8080/TCP,7098/TCP   43h
```